OpenMTRDemo/Forms/ExpandedImageDialog.cs

```diff
@@ -108,31 +121,42 @@ private void SetDisableableControls(bool state)
         private void cannyThreshold1Number_ValueChanged(object sender, EventArgs e)
         {
             CannyThreshold1Slider.Value = (int)CannyThreshold1Number.Value;
-            RenderOutput();
+            Render();
         }
 
         private void cannyThreshold2Number_ValueChanged(object sender, EventArgs e)
         {
             CannyThreshold2Slider.Value = (int)CannyThreshold2Number.Value;
-            RenderOutput();
+            Render();
         }
 
         private void cannyThreshold1Slider_Scroll(object sender, EventArgs e)
         {
             CannyThreshold1Number.Value = CannyThreshold1Slider.Value;
-            RenderOutput();
+            Render();
         }
 
         private void cannyThreshold2Slider_Scroll(object sender, EventArgs e)
         {
             CannyThreshold2Number.Value = CannyThreshold2Slider.Value;
-            RenderOutput();
+            Render();
+        }
+
+        private void cancelButton_Click(object sender, EventArgs e)
+        {
+            Close();
+        }
+
+        private void okButton_Click(object sender, EventArgs e)
+        {
+            DialogResult = DialogResult.OK;
```

Comment:
Unless I have missed something I don't see this used anywhere? Is this for future usage of passing the modified image back into TiledFilterForm? If that is the case remove this and add that functionality in another PR.